pkg/pillar/types/errortime.go

```diff
@@ -12,19 +12,95 @@ import (
 
 // Common error with timestamp
 
+const (
+	//RetryCountWarning indicates to move severity to Warning in case of more retries
+	RetryCountWarning = 10
+	//RetryTimeWarning indicates to move severity to Warning in case of more time spent for retry
+	RetryTimeWarning = time.Hour
+	//RetryCountError indicates to move severity to Error in case of more retries
+	RetryCountError = 20
+	//RetryTimeError indicates to move severity to Error in case of more time spent for retry
+	RetryTimeError = 2 * time.Hour
```

Comment:
a 2x difference between warning and error seems small. 10x sounds more reasonable.